Libraries/Text/Text.windows.js

```diff
@@ -263,6 +275,14 @@ const Text = React.createClass({
     if (this.context.isInAParentText) {
       return <RCTVirtualText {...newProps} />;
     } else {
+      if (Platform.OS === 'windows' && typeof newProps.children === 'string') {
```

Comment:
`this.props.children` can be an array of strings and in many cases this array can contain just one string. If such arrays are concatenated with `.join("")` this fix covers 99% of text in S4L.

Reply:
This was fixed in S4L.